### PR TITLE
EUI-6974 V3 tech ticket for icons

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6974

### Change description ###
Tech ticket for the folder ICON

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html [hunk 1]
```diff
@@ -59,14 +59,14 @@ <h2 class="heading-h2 error-summary-heading" id="edit-case-event_error-summary-h
                       <th id="case-viewer-field-label" *ngIf="!isFieldToHaveNoLabel(field)">
                         <div class="case-viewer-label text-16">{{field.label}}</div>
                       </th>
-                      <td>
+                      <th>
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html:62]: @johnbenjamin-hmcts - why the change from `<td>` to `<th>`? If I remember correctly, the same change made last time caused problems elsewhere and I had to revert it

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    const documentsTreeNodes: DocumentTreeNode[] = [
+      {
+        name: 'Beers encyclopedia'
+      }
+    ];
+    expect(component.getDocuments(documents)).toEqual(documentsTreeNodes);
+  });
+
+  it('should get uncategorised documents', () => {
+    const uncategorisedDocuments: CaseFileViewDocument[] =  [
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:54]: @johnbenjamin-hmcts - can remove extra whitespace after `=`

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      name: 'Uncategorised documents',
+      children: [
+        {
+          name: 'Uncategorised document 1'
+        },
+        {
+          name: 'Uncategorised document 2'
+        }
+      ]
+    }
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts:80]: @johnbenjamin-hmcts - needs a terminating `;`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - a few comments to address, everything else is OK

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html:62]: @Daniel-Lam if `<td>` then the case file view tab content is rendered only a quarter of the screen. Had to change it to `<th>` to fix this issue.

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts (hunk 3 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts]: Thank you @Daniel-Lam I have updated the code.

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts]: Thank you @Daniel-Lam I have updated the code.

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html (hunk 1 above)
Comment (Daniel-Lam) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.html:62]: @johnbenjamin-hmcts - we will need to test with other tabs because I'm sure this caused a problem in the past

**Review by Daniel-Lam — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+    return { name: CaseFileViewFolderComponent.UNCATEGORISED_DOCUMENTS_TITLE, children: documents };
+  }
+
+  public ngOnDestroy(): void {
+    if (this.categoriesAndDocumentsSubscription) {
+      this.categoriesAndDocumentsSubscription.unsubscribe();
+    }
+  }
+
+  public loadCategories(): CaseFileViewCategory[] {
```
Comment (MateiHMCTS) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:84]: Import the test data from the file you've created

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts [hunk 5]
```diff
@@ -1,17 +1,40 @@
...
+import { RouterTestingModule } from '@angular/router/testing';
+import { of } from 'rxjs';
+import { CaseFileViewService } from '../../../services';
 import { CaseFileViewFieldComponent } from './case-file-view-field.component';
+import createSpyObj = jasmine.createSpyObj;
 
 describe('CaseFileViewFieldComponent', () => {
   let component: CaseFileViewFieldComponent;
   let fixture: ComponentFixture<CaseFileViewFieldComponent>;
+  let mockCaseFileViewService: any;
```
Comment (MateiHMCTS) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts:13]: type should be jasmine.SpyObj<CaseFileViewService>

**Review by MateiHMCTS — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts (hunk 4 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts:84]: thank you @MateiHMCTS I will do this as part of the next in progress ticket https://tools.hmcts.net/jira/browse/EUI-5377

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts (hunk 5 above)
Comment (johnbenjamin-hmcts) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts:13]: thank you @MateiHMCTS I will do this as part of the next in progress ticket https://tools.hmcts.net/jira/browse/EUI-5377

Comment (Daniel-Lam) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts:13]: @johnbenjamin-hmcts - I've already corrected this in my PR, so it'll be there when it gets merged
